### DISCUSSION TITLE
E117 command line

### DISCUSSION THREAD

**Discussion by VasylyshynPavlo** [jdhao/nvim-config#339] [Q&A]
Hi! Im new in Linux so i don't know as well as polible
I follow installation guide but when i try to :command i see an error
![2024-09-28_15-39](https://github.com/user-attachments/assets/e1a910b5-f581-4334-b373-71a24f907202)
Can u pls help me?
My OS is Arch Bspwm

**Comment by farhoudkazemi91-spec** [comment]
Hello
